docs/UserGuide.md

```diff
@@ -66,80 +79,124 @@ Example:
         3. Toss the spaghetti in the sauce,
         4. Taste and season with salt, black pepper and chilli flakes.
 
-###### [return to table of content](#table-of-content)
+[<sub>return to table of content<sub>](#table-of-content)
 
-### Delete recipe
+### Delete an existing recipe: `delete`
 Removes a stored recipe based on the number it is associated with in the `list` <br>
 Usage: `delete -x <recipe index>` or `delete <recipe name>` <br> 
 Example:
-- delete -x 1
+- `delete -x 1`
     - This would delete the first entry in the list of recipes.
-- delete Aglio Olio 
+- `delete Aglio Olio` 
     - This would delete the recipe called 'Aglio Olio'.
-  
 
-###### [return to table of content](#table-of-content)
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Deletion of recipes by its name are case-sensitive!**<br>
+To prevent accidental deletions, McKitchen makes it _slightly_ inconvenient for users to delete a recipe by its name by making it case-sensitive. A faster way to remove a recipe from McKitchen is through its index.
+
+</div>
 
-### Find recipe
+[<sub>return to table of content<sub>](#table-of-content)
+
+### Find recipe an existing recipe: `find`
 Searches for a recipe based on the keywords (seperated by comma). <br>
-Usage: `find <keyword> [, other keywords, seperated by comma]` <br>
+Usage: `find <keyword> [other keywords, seperated by commas]` <br>
+
 Example: 
-- find western, fish
-  - This would display all the recipes that are "western", contains "fish", or both.
+- Let's say that you would like to cook a Western cuisine and have spaghetti and garlic on hand. <br>
+        The command that you would execute should look something like this:
+    - `find western, spaghetti, garlic`
 
-###### [return to table of content](#table-of-content)
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Keywords used for the find feature are case-insensitive!**<br>
+Don't worry, McKitchen would still be able to find "Secret Family Sauce" even if you search for "secret family sauce" (or searching for the secret ingredient) :wink:
+
+</div>
 
-### List recipe
+[<sub>return to table of content<sub>](#table-of-content)
+
+### List all stored recipes: `list`
 Displays all the available recipes that has been stored locally <br>
 Usage: `list`
 
-###### [return to table of content](#table-of-content)
+[<sub>return to table of content<sub>](#table-of-content)
 
-### View recipe
+### View recipe an existing recipe: `view`
```

Comment:
Should be "View an exisiting recipe" instead of "View recipe an existing recipe" to be consistent with Delete
```suggestion
### View an existing recipe: `view`
```

Reply:
omg how did i miss those HAHAHA, thanks for catching them!